runtime/vm/stringhelpers.cpp

```diff
@@ -396,41 +332,39 @@ copyFromStringIntoUTF8(J9VMThread * vmThread, j9object_t string, char * dest)
  * @param[in] stringFlags the flag to determine performing '.' --> '/'
  * @param[in] prependStr the string to be prepended before the string object to be copied
  * 				it can't be NULL but can be an empty string ""
+ * @param[in] prependStrLength The length of prependStr as computed by strlen.
  * @param[in] buffer the buffer for the string
  * @param[in] bufferLength the buffer length, not expected to larger than 64K
+ * @param[out] utf8Length If not NULL returns the computed length (in bytes) of the copied UTF8 string in the buffer excluding the NULL terminator.
  *
  * @return a char pointer to the string (with NULL termination)
  */
```

Comment:
Pls remove these function comments, they are in head file `vm_api.h`, not duplicate them here. (asking you do the cleanup since you touched it :)

Reply:
Fixed in 84f8304